website/content/community/_index.md

```diff
@@ -150,6 +150,12 @@ the [github repository](https://github.com/metallb/metallb), and add
 your fork as a remote in `$GOPATH/src/go.universe.tf/metallb`, with
 `git remote add fork git@github.com:<your-github-user>/metallb.git`.
 
+## Commit Messages
+
+The following are our commit message guidelines:
+
+* Line wrap at 72 characters
```

Comment:
Can we also add some guidelines on how the commit message should look like? Even a reference to https://chris.beams.io/posts/git-commit/ would be enough I think.

Reply:
link added - that's a great writeup